layouts/partials/navbar.html

```diff
@@ -0,0 +1,33 @@
+{{ $cover := .HasShortcode "blocks/cover" }}
+<nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
+        <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-lowercase font-weight-medium">{{ .Site.Title }}</span>
```

Comment:
Only difference between this and the original Docsy file is:

`<span class="text-lowercase font-weight-medium">`